test/game_test.rb

```diff
@@ -75,4 +75,87 @@ def test_DEBUG_board_renders_shot_behavior_for_testing
 
   end
 
+  def test_it_has_a_main_menu
+    skip
+    #does print to the screen as expected
+    expected = "Welcome to BATTLESHIP
+Enter p to play. Enter q to quit."
+
+    assert_equal expected, @game.main_menu
+
+  end
+
+  def test_it_has_a_set_up_ships
```

Comment:
Confirming I see test for players_setup_ships

Reply:
Yes, I am writing a method so that within our Game.start we have the menu to set up the ship methods show up in our game class prompting both players to post their ships!

I'm not sure if it's the right call, but it sure makes it easier to have what is being printed onto the terminal so close to the functionality of the method.